src/components/swapv2/PairSuggestion/index.tsx

```diff
@@ -221,6 +221,7 @@ export default forwardRef<PairSuggestionHandle, Props>(function PairSuggestionIn
         search: stringify(newQs),
       })
       setShowModalImportToken(true)
+      refInput.current?.blur()
```

Comment:
Is this included? 🤔

Reply:
I added description.   
![image](https://user-images.githubusercontent.com/33005392/184308831-99ec42df-b9a5-41fb-b398-ce6e7084ac4f.png)